docs/writing-tests/properties.md

```diff
@@ -339,3 +361,34 @@ docs](/inferno-core/docs/Inferno/Entities/TestSuite.html#suite_summary-class_met
 
 **Reference**: [`config` in the API
 docs](/inferno-core/docs/Inferno/DSL/Configurable.html#config-instance_method)
+
+----
+## Reorder
```

Comment:
These aren't runnable properties, so I don't think they need to be documented in this file.